var/spack/repos/builtin/packages/pmt/package.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Pmt(CMakePackage):
+    """A multi-runtime implementation of a distributed merge tree
+    segmentation algorithm. The implementation relies on the framework
+    BabelFlow, which allows to execute the algorithm on different runtime
+    systems."""
+
+    homepage = "https://bitbucket.org/cedmav/parallelmergetree"
+    url      = "https://bitbucket.org/cedmav/parallelmergetree/get/ascent.zip"
```

Comment:
```suggestion
    git      = "https://bitbucket.org/cedmav/parallelmergetree.git"
```
Then you won't need to specify it below.

Reply:
thanks  -- fixed this